### DISCUSSION TITLE
ExFAT Support

### DISCUSSION THREAD

**Discussion by woebags** [tesla-android/issue-tracker#296] [Q&A] [answered]
Hello,

I've been trying to find a way to get files larger than 4GB read by Tesla Android but it seems that ExFAT isn't supported. I believe some Android 13 devices (Samsung and Pixel) do have ExFAT Support. Is this something that is possible in the Android 13 on Tesla Android?

Thank you so much for this project! It's amazing.

**Comment by mikegapinski** [comment, marked as answer]
@woebags 

It seems like it needs to be enabled in the kernel, https://www.esper.io/blog/android-dessert-bites-27-exfat-on-pixel-532176849

Seems like a quick win, thanks for the tip!
